native/src/assets/licenses.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
Why is this new/added?

Reply:
should be `.gitignore`d now. Don't know actually why this worked previously though...